worldguard-bukkit/src/main/java/com/sk89q/worldguard/bukkit/util/report/SchedulerReport.java

```diff
@@ -52,19 +53,22 @@ public Optional<Field> load(Class<?> clazz) throws Exception {
     public SchedulerReport() {
         super("Scheduler");
 
-        List<BukkitTask> tasks = Bukkit.getServer().getScheduler().getPendingTasks();
+        append("Error", "MODDED FOLIA VERSION - DO NOT REPORT TO WORLDGUARD");
```

Comment:
That looks like it's not finished yet? Please don't remove the whole report.

Reply:
Whoops, sorry about that. Originally I wasn’t intending to PR it so I put a comment there so people wouldn’t spam you guys if they found my fork. It seems I forgot to remove that comment.

Reply:
Having looked closer into this, until TechnicallyCoded adds a method to getAllWrappedTasks either active or pending.  I will contact him as he is a work colleague.